client/src/omtLinkProvider.ts

```diff
@@ -0,0 +1,98 @@
+import * as path from 'path';
+import { DocumentLink, DocumentLinkProvider, TextDocument } from 'vscode';
+import { Position, Range, Uri, workspace } from 'vscode';
+import { readFileSync } from 'fs';
+
+const MATCHER = /( +["']?)(.*\.omt)/;
+const importMatch = /^import:/g;
+const otherDeclareMatch = /^(\w+):/g;
+
+// Provides DocumentLinks for the imports of an OMT file.
+export default class OMTLinkProvider implements DocumentLinkProvider {
+
+    public provideDocumentLinks(document: TextDocument): Promise<DocumentLink[]> {
+
+        return Promise.resolve(workspace.findFiles('**/tsconfig**.json').then(
+            (tsConfigs) => {
+                const pathPaths = tsConfigs
+                    // we need to filter for same parent because another config may have some of the same paths
+                    .filter(uri => document.fileName.startsWith(uri.path.substr(0, uri.path.lastIndexOf('/'))))
+                    .map(uri => {
+                        const file = readFileSync(uri.path, 'utf8');
+                        try {
+                            const json = JSON.parse(file);
+                            if (json.compilerOptions && json.compilerOptions.paths) {
+                                // now make a path from the config folder to the keys value and map that
+                                const paths: [string, string][] = [];
+                                for (var key in json.compilerOptions.paths) {
+                                    const relPath = json.compilerOptions.paths[key].toString();
+                                    const newPath = path.resolve(path.dirname(uri.path), relPath);
+                                    paths.push([key.substr(0, key.lastIndexOf('/*')), newPath]);
+                                }
+                                return paths;
+                            }
+                        } catch (e) { console.error(); }
+                        // all other paths return undefined
+                    }).filter(paths => paths != undefined);
+                //make it typed
```

Comment:
space

Reply:
what do you mean?

Reply:
There is no space between the slashes and the comment:
```suggestion
                // make it typed
```